pkgs/tools/misc/fortune/default.nix

```diff
@@ -1,35 +1,21 @@
 { stdenv, fetchurl, cmake, recode, perl }:
 
-let srcs = {
-      fortune = fetchurl {
-        url = "https://github.com/shlomif/fortune-mod/archive/fortune-mod-${version}.tar.gz";
-        sha256 = "89223bb649ea62b030527f181539182d6a17a1a43b0cc499a52732b839f7b691";
-      };
-      shlomifCommon = fetchurl {
-        url = https://bitbucket.org/shlomif/shlomif-cmake-modules/raw/default/shlomif-cmake-modules/Shlomif_Common.cmake;
-        sha256 = "62f188a9f1b7ab0e757eb0bc6540d9c0026d75edc7acc1c3cdf7438871d0a94f";
-      };
-    };
-    version = "2.6.2";
-in
-stdenv.mkDerivation {
-  name = "fortune-mod-${version}";
+stdenv.mkDerivation rec {
+  pname = "fortune-mod";
+  version = "2.10.0";
 
-  src = srcs.fortune;
-
-  sourceRoot = "fortune-mod-fortune-mod-${version}/fortune-mod";
+  src = fetchurl {
+    url = "https://www.shlomifish.org/open-source/projects/fortune-mod/arcs/fortune-mod-${version}.tar.xz";
+    sha256 = "07g50hij87jb7m40pkvgd47qfvv4s805lwiz79jbqcxzd7zdyax7";
+  };
 
   nativeBuildInputs = [ cmake perl ];
 
   buildInputs = [ recode ];
 
-  preConfigure = ''
-    cp ${srcs.shlomifCommon} cmake/Shlomif_Common.cmake
-  '';
-
   postInstall = ''
-    mv $out/games/fortune $out/bin/fortune
-    rm -r $out/games
+    mv -v $out/games/fortune $out/bin/fortune
```

Comment:
I'm seeing the output directories like
```
local
│   └── share
│       └── games
```

Probably shouldn't have the `local`.